cypress/support/fragments/settings/remote-storage/configurations.js

```diff
@@ -27,7 +27,9 @@ const dataSynchronizationSettingsAccordion = Accordion('Data synchronization set
 const saveAndCloseBtn = Button('Save & close');
 const saveBtn = Button('Save');
 const cancelButton = Button('Cancel');
+const deletelButton = Button('Delete');
 const confirmationModal = Modal({ id: 'save-confirmation-modal' });
+const deleteConfirmationModal = Modal({ title: including('Remove ') });
```

Comment:
Please create separate module for confirmation modal and move all logic to it.

Here is the example: cypress/support/fragments/orders/modals/createInvoiceModal.js

Reply:
done